static/app/views/performance/trends/changeExplorerUtils/metricsTable.tsx

```diff
@@ -0,0 +1,415 @@
+import {ReactNode, ReactText, useMemo} from 'react';
+import {Location} from 'history';
+import moment from 'moment';
+
+import GridEditable, {
+  COL_WIDTH_UNDEFINED,
+  GridColumnOrder,
+} from 'sentry/components/gridEditable';
+import SortLink from 'sentry/components/gridEditable/sortLink';
+import {t} from 'sentry/locale';
+import {Organization} from 'sentry/types';
+import {parsePeriodToHours} from 'sentry/utils/dates';
+import {TableData, useDiscoverQuery} from 'sentry/utils/discover/discoverQuery';
+import EventView from 'sentry/utils/discover/eventView';
+import {
+  AggregationKeyWithAlias,
+  ColumnType,
+  fieldAlignment,
+  QueryFieldValue,
+} from 'sentry/utils/discover/fields';
+import {Container} from 'sentry/utils/discover/styles';
+import {DiscoverDatasets} from 'sentry/utils/discover/types';
+import {formatPercentage} from 'sentry/utils/formatters';
+import theme from 'sentry/utils/theme';
+import {MutableSearch} from 'sentry/utils/tokenizeSearch';
+import {TransactionThresholdMetric} from 'sentry/views/performance/transactionSummary/transactionThresholdModal';
+import {ExplorerText} from 'sentry/views/performance/trends/changeExplorer';
+import {
+  NormalizedTrendsTransaction,
+  TrendFunctionField,
+  TrendsTransaction,
+  TrendView,
+} from 'sentry/views/performance/trends/types';
+
+type MetricsTableProps = {
+  isLoading: boolean;
+  location: Location;
+  organization: Organization;
+  transaction: NormalizedTrendsTransaction;
+  trendFunction: string;
+  trendView: TrendView;
+};
+
+const fieldsNeeded: AggregationKeyWithAlias[] = ['tps', 'p50', 'p95'];
+
+export function MetricsTable(props: MetricsTableProps) {
+  const {trendFunction, transaction, trendView, organization, location, isLoading} =
+    props;
+  let p50: TableDataRow | undefined;
+  let p95: TableDataRow | undefined;
+
+  if (trendFunction === TrendFunctionField.P50) {
+    p50 = getTrendsRowData(transaction, TrendFunctionField.P50);
+  }
+
+  if (trendFunction === TrendFunctionField.P95) {
+    p95 = getTrendsRowData(transaction, TrendFunctionField.P95);
+  }
```

Comment:
I don't think these need to be their own `let` + if conditions, you can probably put them on the otherside of the ternary eg. `p95Events` below